config/locales/redlands_chapters_and_book_section.en.yml

```diff
@@ -1,8 +1,12 @@
-en:
+en-REDLANDS:
```

Comment:
I think this should stay as `en` since it is for the work type and not just the `REDLANDS` locale.  If it is supposed to only be for the `REDLANDS` locale then I think you should make a new file which matches the convention (`redlands_chapters_and_book_section.en-REDLANDS.yml`) and make the change there instead.